policy/modules/services/xserver.if

```diff
@@ -146,6 +146,7 @@ interface(`xserver_role',`
 
 	# Communicate via System V shared memory.
 	allow $2 xserver_t:shm rw_shm_perms;
+	allow $2 xserver_tmp_t:file rw_file_perms;
```

Comment:
What errors do you get if you don't allow this?

I'd like to determine if this makes sense with the other shm accesses, or if it should go somewhere else in the interface.